packages/forms-web-app/__tests__/unit/controllers/examination/submission-error/controller.test.js

```diff
@@ -0,0 +1,78 @@
+const {
```

Comment:
The error page is never triggered. 

We need to determine if this page is displayed for the whole examination journey. 

If so we would need to redirect to this page in all the catch blocks of our controllers in the examination journey.